src/panels/lovelace/editor/view-editor/hui-view-background-editor.ts

```diff
@@ -43,19 +35,17 @@ export class HuiViewBackgroundEditor extends LitElement {
           "ui.panel.lovelace.editor.edit_view.background.title"
         )}
       </p>
-      <ha-picture-upload
+      <ha-selector-image
```

Comment:
```suggestion
      <ha-selector-image
        .label=${this.hass.localize(
          "ui.panel.lovelace.editor.edit_view.background.title"
        )}
```

And remove the `p` label to avoid duplicate label like this 

![CleanShot 2024-05-29 at 15 16 20](https://github.com/home-assistant/frontend/assets/5878303/518df1df-99ec-4934-9cbc-1e529e2e693d)